runbot_travis2docker/demo/runbot_repo_demo.xml

```diff
@@ -3,7 +3,7 @@
     <data>
 
         <record id="runbot_repo_demo1" model="runbot.repo">
-            <field name="name">github.com/vauxoo-dev/fast-travis-test.git</field>
+            <field name="name">https://github.com/vauxoo-dev/fast-travis-test.git</field>
```

Comment:
We are using the original repo without the following patch: https://github.com/odoo/odoo-extra/pull/74